### PR TITLE
Feature/form validation

### PR DESCRIPTION
**Pull request by walawe**

### CODE REVIEW

File: src/sidebar-items.json [hunk 1]
```diff
@@ -169,6 +169,17 @@
     "url": "form-layout.html",
     "icon": "file-earmark-medical-fill"
   },
+  {
+    "name": "Form Validation",
+    "icon": "pen-fill",
```
Comment (zuramai) [inline src/sidebar-items.json:174]: Can you change the icon to something like bootstrap icon's `journal-check`?

**Review by zuramai — CHANGES_REQUESTED** [review]
Change the icon and it would have a better look

File: src/sidebar-items.json (hunk 1 above)
Comment (bagussatoto) [reply src/sidebar-items.json:174]: real

**Review by bagussatoto — APPROVED** [review]
# are there any feature changes?

**Review by zuramai — APPROVED** [review]
Great!

**Pull request merged**